pkgs/epnix/support/devlib2/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  epnixLib,
+  mkEpicsPackage,
+  fetchFromGitHub,
+  fetchpatch,
+  local_config_site ? {},
+  local_release ? {},
+}:
+mkEpicsPackage rec {
+  pname = "devlib2";
+  version = "2.12";
+  varname = "DEVLIB2";
+  doCheck = false;
+
+  inherit local_config_site local_release;
+
+  src = fetchFromGitHub {
+    owner = "epics-modules";
+    repo = "devlib2";
+    rev = version;
+    sha256 = "sha256-5rjilz+FO6ZM+Hn7AVwyFG2WWBoBUQA4WW5OHhhdXw4=";
+  };
+
+  meta = {
+    description = "devLib2 - Library for direct MMIO access to PCI and VME64x";
+    homepage = "https://github.com/epics-modules/devlib2";
+    license = lib.licenses.lgpl21Plus;
```

Comment:
devlib2 seems to be developed under the EPICS license: https://github.com/epics-modules/devlib2/blob/master/LICENSE